### PR TITLE
fix: Added support for django42

### PR DESCRIPTION
**Pull request by zubairshakoorarbisoft**
## Description

Description of changes made

## Ticket Link

Link to the associated ticket

## Post-review

* [ ] Squash commits into discrete sets of changes
* [ ] Ensure that once the changes have been [deployed to stage](https://gocd.tools.edx.org/go/pipeline/activity/stage-enterprise_catalog), prod is manually deployed

### CODE REVIEW

File: enterprise_catalog/settings/base.py [hunk 1]
```diff
@@ -369,7 +369,11 @@
 }
 SESSION_EXPIRE_AT_BROWSER_CLOSE = False
 SOCIAL_AUTH_REDIRECT_IS_HTTPS = False
-STATICFILES_STORAGE = 'django.contrib.staticfiles.storage.StaticFilesStorage'
+STORAGES = {
```
Comment (awais786) [inline enterprise_catalog/settings/base.py]: Revert this change.

**Review by awais786 — APPROVED** [review]

**Review by iloveagent57 — APPROVED** [review]
